schemas/constants.yml

```diff
@@ -136,23 +136,31 @@ dependency-relation:
 
 # Description of actions. They are defined on the provisioner
 action-description: |
-  Actions are defined on the provisioner and they are performed via a request using the `url` and `method`.
-  Depending on the `context`, the `url` will be filled with parameters. In addition, `context` is
-  used by the front-end to know where to display the action. For example, `context=worker`
-  will display the action on the worker explorer.
-
-  _Note: The request will be signed with the user's Taskcluster credentials._
-
-# Description of the action context
-action-context-description: |
-  Actions have a "context" that is one of provisioner, worker-type,
-  or worker, indicating which it applies to. `context` is used by the front-end to know where to display the action.
-
-  | `context`   | Page displayed        |
-  |-------------|-----------------------|
-  | provisioner | Provisioner Explorer  |
-  | worker-type | Workers Explorer      |
-  | worker      | Worker Explorer       |
+  Actions provide a generic mechanism to expose additional features of a
+  provisioner, worker type, or worker to taskcluster clients.
```

Comment:
It might be better to just include one paragraph here, and link to https://docs.taskcluster.net/reference/platform/taskcluster-queue/docs/actions, as in `actions.yml`.